### PR TITLE
Bump dependencies versions

### PR DESCRIPTION
**Pull request by fflorent**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -86,14 +86,14 @@
...
-    "catw": "1.0.1",
     "chai": "4.2.0",
     "chai-as-promised": "7.1.1",
     "chance": "1.0.16",
+    "chokidar-cli": "^3.0.0",
     "esbuild-loader": "2.19.0",
     "eslint": "8.18.0",
     "http-proxy": "1.18.1",
-    "i18next-scanner": "4.1.0",
+    "i18next-scanner": "^4.4.0",
```
Comment (berhalak) [inline package.json:96]: Can you use explicit versions numbers (here without `^`). Sometimes it is hard to track down the bug due to mismatch in local and prod version of some dependency.

Comment (fflorent) [reply package.json]: Done!

File: app/client/ui/HomeImports.ts [hunk 2]
```diff
@@ -39,9 +40,9 @@ export async function fileImport(
     const timezone = await guessTimezone();
 
     if (workspaceId === "unsaved") {
-      function onUploadProgress(ev: ProgressEvent) {
-        if (ev.lengthComputable) {
-          progress.setUploadProgress(ev.loaded / ev.total * 100);   // percentage complete
+      function onUploadProgress(ev: AxiosProgressEvent) {
+        if (ev.event.lengthComputable) {
```
Comment (fflorent) [inline app/client/ui/HomeImports.ts:44]: FWIW, I achieved to hit a breakpoint here. The `workspaceId` equals to `"unsaved"` when the user is not logged in but is allowed to upload documents (temporary).

The values are the one that are expected.

File: sandbox/watch.sh [hunk 3]
```diff
@@ -16,7 +16,9 @@ if [ ! -e _build ]; then
 fi
 
 tsc --build -w --preserveWatchOutput $PROJECT &
-catw app/client/*.css app/client/*/*.css -o static/bundle.css -v & webpack --config $WEBPACK_CONFIG --mode development --watch &
+css_files="app/client/**/*.css"
+chokidar "${css_files}" -c "bash -O globstar -c 'cat ${css_files} > static/bundle.css'" &
+webpack --config $WEBPACK_CONFIG --mode development --watch &
```
Comment (jordigh) [inline sandbox/watch.sh:21]: This is a pretty cool way to replace the abandoned `catw` package.

**Review by jordigh — APPROVED** [review]

**Comment by fflorent** [comment]
@jordigh Thanks for your review. Do you wait for another review before merging?

Otherwise I do not have the rights to merge myself.

**Comment by jordigh** [comment]
@fflorent I need to first verify deploying to our own Grist SAAS before I merge.

**Comment by fflorent** [comment]
@jordigh Sounds good! :+1:

**Comment by jordigh** [comment]
Thanks for your patience. After some internal testing, we are satisfied with the state of this PR.

**Pull request merged**